generated/source-repo/githubactions/.github/workflows/build-and-update-gitops.yml

```diff
@@ -87,43 +89,44 @@ jobs:
       with:
         fetch-depth: '2'
     - name: Pre-init
-      run: | 
-        echo "Using quay.io/redhat-appstudio/dance-bootstrap-app:rhtap-runner"
-        mkdir -p out
-        docker run -v $(pwd)/out:/out quay.io/redhat-appstudio/dance-bootstrap-app:rhtap-runner bash /work/copy-scripts.sh
```

Comment:
Not a blocker, but we could remove the copy-scripts.sh script and the related RUN command in the Dockerfile in this PR also.